python/grass/pygrass/vector/table.py

```diff
@@ -955,11 +955,16 @@ def by_index(self, indx):
         """
         nlinks = self.num_dblinks()
         if nlinks == 0:
-            raise IndexError
+            raise IndexError(_("No links available."))
```

Comment:
I'm surprised that none of the flake8-errmsg rules kicked in.
Something like [EM102](https://docs.astral.sh/ruff/rules/f-string-in-exception/), [EM101](https://docs.astral.sh/ruff/rules/raw-string-in-exception/#legend), or [EM103](https://docs.astral.sh/ruff/rules/dot-format-in-exception/).

Rereading them, it seems they only apply when the formatting is done directly, not when done indirectly by having another function called ( our `_()` for translation).

If you feel that the spirit of the EM10* rules and the examples of the tracebacks created when using the patterns suggested in EM10*, consider applying it here too.

Reply:
Thanks, applying EM102 in [2aeb685](https://github.com/OSGeo/grass/commit/2aeb685b0a3be6bf83259f4b0fdb8b50ed6961b9)